### PR TITLE
Add celo-org/gitcoin repo

### PR DESCRIPTION
**Pull request by critesjosh**
Add celo-org/gitcoin repo

### CODE REVIEW

**Comment by jubos** [comment]
Thanks!

**Comment by Chethan-4255** [comment]
Add celo-org/gitcoin repo

**Comment by critesjosh** [comment]
> https://github.com/lexycole/My-DApp

Are you trying to add this repo to the report?

**Comment by lexycole** [comment]
trying to merge with Electric-captical, but is is a done deal.
Thanks

File: data/ecosystems/c/celo.toml [hunk 1]
```diff
@@ -188,6 +188,9 @@ url = "https://github.com/celo-org/gcloud-kubectl-docker"
 [[repo]]
 url = "https://github.com/celo-org/geth_exporter"
```
Comment (lexycole) [inline data/ecosystems/c/celo.toml:190]: [[repo]]
url = "https://github.com/lexycole/My-DApp"

**Review by lexycole — COMMENTED** [review]
[[repo]]
url = "https://github.com/lexycole/My-DApp"

File: data/ecosystems/c/celo.toml (hunk 1 above)
Comment (lexycole) [inline data/ecosystems/c/celo.toml:190]: [[repo]]
url = "https://github.com/lexycole/My-DApp"

**Review by lexycole — COMMENTED** [review]
[[repo]]
url = "https://github.com/lexycole/My-DApp"

**Comment by lexycole** [comment]
> @lexycole you need to edit [this file](https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/c/celo.toml) and open a pull request to add additional repos

Thanks @critesjosh 
My apology :)

**Comment by lexycole** [comment]
Got it.
Thanks

**Pull request merged**